### DISCUSSION TITLE
2 Algolia-questions: First-time-sync; geo-location

### DISCUSSION THREAD

**Discussion by webwurm** [AntlerVC/firetable#966] [Support / Q&A] [answered]
I integrated Algolia now via extension like [here]([url]https://docs.rowy.io/extensions/algolia-index(url)) - and it works perfectly.
Yet I have 2 questions:

1. What is the best way to sync the current list of documents? Every newly created syncs fine, but how do I do the "initial-sync", to bring all my documents to Algolia? Is there a best-practice?

2. Searching by geo-locaction: Algolia expects a geo-location like this:

```
"_geoloc": {
  "lat": 40.639751,
  "lng": -73.778925
}
```

Rowy delivers like this (from the geopoint-field):
```
  "_geoloc": {
    "_latitude": 48.2027517,
    "_longitude": 16.3468288
  }
```

Should I build the correct format with a derivative-field that outputs a string - or what is the best way to do it here?

Thanks a lot - I hope the answers to this questions also help others!

**Comment by BeeBombshell** [comment, marked as answer]
Hi @webwurm if this is still an issue, perhaps you can join our upcoming [office hours](http://rowy.io/events) or book a [short 1:1 meeting](https://rowy.io/meet) so that we can go over this and get it resolved.

**Reply by BeeBombshell** [reply]
@webwurm checked the above links that were broken and fixed them now. Hope you'll have your issue resolved now...

**Reply by webwurm** [reply]
Thanks, I'll give it a try!
